### ISSUE TITLE
Simplify Dialect/Control/Layout

### ISSUE THREAD

**Issue by roll** [frictionlessdata/frictionless-py#947]
# Overview

We need to simplify it (and add the `code` property? - see below)

---

Consider writing `Table Dialect` spec.

---

Consider supporting skipping rows at the end of the file - https://frictionlessdata.slack.com/archives/C0362US1U3G/p1651741302921959

---

Currently, almost all the descriptors that have different implementations like Types/Steps/Checks/etc have the "code" property in the descriptor for identification. It seems Dialect/Control miss it. It should be like:

```python
class CsvDialect:
    code = 'csv'
```

```yaml
code: csv
delimiter: ,
# other props
```

---

Handle - https://github.com/frictionlessdata/frictionless-py/issues/1076

**Comment by jze** [comment]
It seems to be common for official agencies add some text (hints, copyright statements etc.) in the last rows of a CSV file. Over time, the number of data lines in the CSV file grows. Therefore, you cannot specify a fixed number of relevant lines. It would be useful if you could ignore a fixed number of lines at the end.

**Comment by roll** [comment]
Thanks @jze, totally makes sense

**Comment by roll** [comment]
FIXED in #944
